### ISSUE TITLE
Allow downscaling shaders

### ISSUE THREAD

**Issue by lunarfyre7** [y4my4my4m/kde-shader-wallpaper#23]
It would be size to render the shaders at a lower resolution than the screen size. In my case for example I am using a 4k monitor with a fairly modest gpu, so a lot of the more intensive shaders stutter.

**Comment by y4my4my4m** [comment]
@lunarfyre7 
yes, ive considered this at the moment i'm refactoring the whole code and splitting the settings/customizations into separate windows.

In the meantime you could probably achieve the result you want by modifying:
`~/.local/share/plasma/wallpapers/online.knowmad.shaderwallpaper/contents/ui/Components/ShaderEngine.qml` 

Around the startCode and/or vertexShader parts.

**Comment by y4my4my4m** [comment]
@pyrareae 5 years later, this is now implemented in the V4 (current master branch)
